dashboard/app/controllers/potential_teachers_controller.rb

```diff
@@ -4,13 +4,16 @@ class PotentialTeachersController < ApplicationController
 
   # url: /potential_teachers METHOD: POST
   def create
-    @potential_teacher = PotentialTeacher.new(potential_teacher_params)
-    begin
-      @potential_teacher.save!
-      render json: {message: 'Potential teacher created successfully'}, status: :created
-    rescue ActiveRecord::RecordInvalid => exception
-      raise "ERROR: #{exception.message}"
+    unless current_user
```

Comment:
My brain struggles with the 'unless' keyword, but ruby complains if I say something like "if !current_user"

Reply:
does it complain as much if you use something like `if current_user.nil?` ?

In general, I agree with you. I like `unless` for inline conditionals but once it's more complex (ie I can't literally say it in a sentence), it hurts my brain

Reply:
(also not suggesting you should actually change this unless you want to. This looks correct)

Reply:
This is a solid alternative. I used 'nil?'. Thanks!